### PR TITLE
get random potential partners, closes #186, closes #188

### LINKED ISSUE #186: Return 10 random users. Just to get things started.

### LINKED ISSUE #188: Return all the users which can be selected by everything which is not flag and not GEO related (e.g. query for female and height, etc.)

### PR DESCRIPTION
**Pull request by briankariuki**

### CODE REVIEW

File: lib/animina_web/potential-partner.ex [hunk 1]
```diff
@@ -0,0 +1,20 @@
+defmodule AniminaWeb.PotentialPartner do
```
Comment (wintermeyer) [inline lib/animina_web/potential-partner.ex]: Shouldn't be the filename `potential_partner.ex`and not `potential-partner.ex`? I am not a 100% sure but `_` feels better to me.

File: lib/animina_web/potential-partner.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  alias Animina.Accounts
+  alias Animina.Accounts.User
+
+  require Ash.Query
+
+  @doc """
+  Gets random users by the given limit.
+  """
+  def get_random_users(limit \\ 10) do
```
Comment (wintermeyer) [inline lib/animina_web/potential-partner.ex]: We should be able to call this function with a `current_user` argument to make sure that this user is not part of the result.

File: lib/animina/accounts/resources/user.ex [hunk 3]
```diff
@@ -104,6 +106,12 @@ defmodule Animina.Accounts.User do
 
   actions do
     defaults [:create, :read, :update]
+
+    read :random_users do
```
Comment (wintermeyer) [inline lib/animina/accounts/resources/user.ex]: Why do we need this action?

**Review by wintermeyer — CHANGES_REQUESTED** [review]

File: lib/animina/accounts/resources/user.ex (hunk 3 above)
Comment (briankariuki) [reply lib/animina/accounts/resources/user.ex]: I've moved this to the PotentialPartner module. Its now part of the query and not an action

File: lib/animina_web/potential-partner.ex (hunk 2 above)
Comment (briankariuki) [reply lib/animina_web/potential-partner.ex]: The function now accepts a user as an argument

**Pull request merged**